module/utils/useItems.js

```diff
@@ -0,0 +1,49 @@
+Hooks.on("renderChatMessage", (message, html, data) => {
+    setTimeout(() => {
+        $(html).find(".reduce-item-count").on("click", (event) => useItem(event));
+    }, 500);
+});
+
+export async function useItem(event){
+    //prevent the default action of the button
+    event.preventDefault();
+
+    // Get the item ID and name from the button element's data-item-id and data-item-name attributes
+    let itemId = event.currentTarget.dataset.item;
+    let itemName = event.currentTarget.dataset.itemName;
+    let parentId = `${event.currentTarget.dataset.itemParentid}`; //convert parentId to string
```

Comment:
```suggestion
    const itemId = event.currentTarget.dataset.item;
    const itemName = event.currentTarget.dataset.itemName;
    const parentId = ""+event.currentTarget.dataset.itemParentid; //convert parentId to string
```
Why the conversion to string... is it not already a string?
Also ""+var or var+"" is a faster cast.